Source/Magick.NET/Shared/Pixels/PixelMappings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright 2013-2018 Dirk Lemstra <https://github.com/dlemstra/Magick.NET/>
+//
+// Licensed under the ImageMagick License (the "License"); you may not use this file except in
+// compliance with the License. You may obtain a copy of the License at
+//
+//   https://www.imagemagick.org/script/license.php
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+// either express or implied. See the License for the specific language governing permissions
+// and limitations under the License.
+
+namespace ImageMagick.Shared.Pixels
```

Comment:
Can you change the namespace to ImageMagick intead? I prefer to have most of the types in that namespace. And can you move the file to the `Source\Magick.NET\Shared\Enums` folder where the rest of the enumerations are?